crates/bevy_ecs/src/system/exclusive_function_system.rs

```diff
@@ -30,6 +30,18 @@ where
     marker: PhantomData<fn() -> Marker>,
 }
 
+impl<Marker, F> ExclusiveFunctionSystem<Marker, F>
+where
+    F: ExclusiveSystemParamFunction<Marker>,
+{
+    /// Return this system with a new name.
+    /// Useful to give closure systems more readable and unique names in debugging and tracing.
```

Comment:
Nit: keep a blank line between the basic description and the detailed docs so it shows up nicer in IDE tooltips.

Reply:
(also applies to the other instances of this doc comment)

Reply:
(also applies to the other instances of this doc comment)